Jenkinsfile

```diff
@@ -298,6 +298,19 @@ pipeline {
       }
       notifyBuildResult()
     }
+    aborted {
```

Comment:
we are overengineering to resolve an issue that is not here if it is something that we will remove as soon as the networks issues disappear I am ok with it, but the network issue is there and will appear on other steps

Reply:
also, it would not bee the same build, the commit can change

Reply:
this gives me evil ideas to recover from known failures 😈

Reply:
Yep, I just wanted to see whether this particular approach in real. I do not like it though

Reply:
> also, it would not bee the same build, the commit can change

Actually that's not really an issue though as the cancel running old builds will be in place anyway. I initially thought it could be potentially an issue but not in our case as far as I'm concerned. 

> this gives me evil ideas to recover from known failures 😈

I initially thought to create a step to help with this, but the only issue that I could not solve is the one regarding the parameters when using the build step, if we could have a way to dynamically know what parameters are available for the given `JOB_NAME` then we could encapsulate the logic in a shared step.